packages/fxa-content-server/app/scripts/views/pair/index.js

```diff
@@ -93,19 +93,24 @@ class PairIndexView extends FormView {
     });
 
     if (needsMobileConfirmed) {
-      this.render();
+      await this.render();
```

Comment:
Not sure why, but doing this in the `afterRender` method caused the `handleRadioEngage` method to not run when the event occurred. I noticed `render` ends up returning a promise, so I tried this and it ended up working fine for our needs.